### PR TITLE
Added build tools and static analysis back

### PR DESCRIPTION
**Pull request by etmarx**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -12,14 +17,20 @@ node {
           stage('Clean') {
               sh 'mvn clean'
           }
-          stage('Build') {
-              sh 'mvn --threads 1C deploy'
+          if (env.BRANCH_NAME.equals('3.2.x')) {
```
Comment (vasanthsubram) [inline Jenkinsfile:20]: 3.2.x is hardcoded. Is that correct ?

File: Jenkinsfile.release [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+node {
+    properties([buildDiscarder(logRotator(daysToKeepStr: '14')), disableConcurrentBuilds()])
+    slackSend (message: "RELEASE was requested, confirm on Jenkins to proceed: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.BUILD_URL})")
+    input 'Are you sure you want to make a release?'
+    slackSend (message: "STARTED RELEASE: Job '${env.JOB_NAME} [${env.BUILD_NUMBER}]' (${env.BUILD_URL})")
+    try {
+        cleanWs()
+        checkout scm
+
+        withMaven(jdk: '8', maven: '3.2.x', mavenSettingsConfig: 'MAVEN_SETTINGS', mavenOpts: '-Xmx8g') {
```
Comment (vasanthsubram) [inline Jenkinsfile.release:11]: 3.2.x is hardcoded. Is that correct ?

Comment (etmarx) [reply Jenkinsfile.release:11]: this is the maven version

File: Jenkinsfile (hunk 1 above)
Comment (etmarx) [reply Jenkinsfile:20]: yes, this will be hard-coded in 3.2.x branch and main

**Pull request merged**